bio_files_processor.py

```diff
@@ -0,0 +1,81 @@
+def convert_multiline_fasta_to_oneline(input_fasta, output_fasta=None):
+    '''
+    Converts multiline FASTA-file to ineline.
+
+    Parametrs
+    ---------------------
+    input_file : file
+        File in .fasta format
+
+    Returns
+    ---------------------
+    output_file : file
+        File in .fasta format.
+        If file not provided, input_file is modified and returned.
+
+    '''
+    with open(input_fasta, 'r') as original_input:
+        current_sequence = ''
+        for line in original_input:
+            if line.startswith('>'):
+                current_sequence += '\n' + line
+            elif not line.startswith('>'):
+                current_sequence += line.strip()

```

Comment:
строка либо начинается с `>`, либо нет))
```suggestion
            if line.startswith('>'):
                current_sequence += '\n' + line
            else:
                current_sequence += line.strip()
```